src/include/stir/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.h

```diff
@@ -131,6 +133,18 @@ typedef RegisteredParsingObject<PoissonLogLikelihoodWithLinearModelForMeanAndLis
 
   void
     add_view_seg_to_sensitivity(const ViewSegmentNumbers& view_seg_nums) const;
+
+
+  std::vector<Bin>  record_cache;//(new std::vector<CListEventROOT>(0));
+  std::vector<float> additive_cache;
+  //! The size of a single
+  unsigned long int cache_size;
+
+  bool cache_lm_file;
+
+  bool recompute_cache;
+  //! The additive sinogram will not be read in memory
+  bool long_axial_fov;
```

Comment:
Any example values that can be added to a `example/sample`. Currently the interaction between these values is not clear.

Reply:
Of course.

Reply:
Modified the ```run_test_listmode_recon.sh``` to test the new functionality and added some doxygen lines.